mercury/views/events.py

```diff
@@ -48,13 +50,152 @@ def delete_event(request, event_uuid=None):
     return redirect("/events")
 
 
+def export_all_event(request):
+    if request.path.__contains__("json"):
```

Comment:
Optional: Use `"json" in request.path` if there's no specific reason.
https://stackoverflow.com/a/1964949